image/verify/src/verifier.rs

```diff
@@ -760,15 +835,18 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
     /// If anti-rollback is disabled, the effective fuse-SVN is zero.
     /// Otherwise, it is SVN-fuses.
     fn effective_fuse_svn(fuse_svn: u32, anti_rollback_disable: bool) -> u32 {
-        if anti_rollback_disable {
+        if cfi_launder(anti_rollback_disable) {
+            cfi_assert!(anti_rollback_disable);
             0_u32
         } else {
+            cfi_assert!(!anti_rollback_disable);
             fuse_svn
         }
     }
 }
 
 #[cfg(all(test, target_family = "unix"))]
+#[cfg(feature = "no-cfi")]
```

Comment:
This is dangerous. With this change, `cargo test` no longer runs these tests.

Reply:
I'm not sure we have a lot of options to fix this. The first one that comes to mind is to define CFI_STATE_ORG and cfi_panic_handler